src/core/common/jsonprovider/jsonprovider.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CORE_COMMON_JSONPROVIDER_JSONPROVIDER_HPP_
```

Comment:
This itf will be used on SM smclient as well. So it should be places in:

core/common/nodeconfig/itf/jsonprovider.hpp under aos::nodeconfig namespace